packages/my-react-package/src/MetaWallet.tsx

```diff
@@ -39,7 +42,7 @@ class MetaWallet {
                 localStorage.setItem('webwalletPrivateKey', newWebWallet.privateKey);
             }
         }
-        console.log('ww',localStorage.getItem('webwalletPrivateKey'));
+        console.log('ww', localStorage.getItem('webwalletPrivateKey'));
```

Comment:
As Madhavan suggested, let's use a logger instead. And in general not log sensitive information (I understand this was for debugging but try to delete such code before pushing).